docs/content/guides/dagster/troubleshooting.mdx

```diff
@@ -0,0 +1,120 @@
+---
+title: Troubleshooting with Dagster | Dagster Docs
+---
+
+# Troubleshooting with Dagster 
```

Comment:
🔧 I think this title should be a bit more specific. Maybe something like `Troubleshooting common issues`